src/main/java/com/example/taxiservicespring/controller/CarController.java

```diff
@@ -0,0 +1,52 @@
+package com.example.taxiservicespring.controller;
+
+import java.util.List;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.taxiservicespring.controller.dto.CarDto;
+import com.example.taxiservicespring.controller.dto.CarModelDto;
+import com.example.taxiservicespring.controller.dto.CategoryDto;
+import com.example.taxiservicespring.service.CarService;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class CarController {
+    private final CarService carService;
+
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping(value = "/car/id/{id}")
```

Comment:
Please use @RequestMapping("/car") instead of duplicate the same url in each method